Commands/grafana/_list-available-plugin.md

```diff
@@ -0,0 +1,9 @@
+# [Command] _grafana list-available-plugin_
+
+list all the available plugins
```

Comment:
Can we organize this command in a new 'az grafana plugins' subgroup? So something like 'az grafana plugins fetch' for this command

Reply:
Never mind, this is already in effect, so don't want to create a breaking change at this point